src/css/options.css

```diff
@@ -35,18 +39,62 @@ body {
 }
 
 .row {
-    padding: 1rem;
-}
-
-.row-label {
-    padding-top: 5px;
+    padding: 0.8rem;
 }
 
 .theme {
-    border-radius: 100%;
+    width: 23px;
+    height: 23px;
+    border-radius: 3px;
+    border: 1px solid #0000006e;
+    cursor: pointer;
+    margin-right: 5px;
 }
 
-.theme:active,.theme:focus
-{
+.theme:active,.theme:focus {
     border: 3px solid #fff;
 }
+
+.save-btn{
+    float: right;
+}
+
+.blue-container{
+    background-color: #0084ff;
+    padding: 30px 30px 30px;
+    box-shadow: 2px 2px 5px rgba(0,0,0,0.5);
+    margin-bottom: 5px;
+    padding-bottom: 10px;
+    padding-top: 10px;
```

Comment:
You can add this padding-bottom and padding-top in the padding style on line no. 64